wondrous-app/utils/helpers.ts

```diff
@@ -188,3 +188,16 @@ export const CHAIN_REGEX = {
   HARMONY: /^one[a-zA-Z0-9]{39}$/gm,
   ETHEREUM: /(^0x[a-zA-Z0-9]{40}$)|(.*\.eth$)/gm,
 };
+
+export const getAddressToDisplay = (address: string): string => {
+  if (!address) {
```

Comment:
there's now two of thes same functions, make sure to remove the other one and update all the references